### ISSUE TITLE
[MAS1.4.1] [Visual Requirements-Welcome] Underline or chevron is not provided for links present in the welcome tab screen 

### ISSUE THREAD

**Issue by awalia13** [microsoft/BotFramework-Emulator#1712]
**Actual Result:**
Underline or chevron is not provided for links present in the welcome tab screen 
Design guidelines, Download command Line tools, From azure, Locally links …etc 

**Expected Result:**
Underline or chevron should be provided for links present in the welcome tab screen 
Design guidelines, Download command Line tools, From azure, Locally links …etc 

**Note:** 
This issue exits for throughout application for all links wherever link present.

Same issue repro for below path 
Navigate to the Open bot ->Select the Open bot control->Open bot dialog
1.Learn more 
Welcome bot Framework emulator ->Navigate to the Sign in with Azure account -> Sign in with Azure account dialog 
1.Dont have an azure account?signup
2.Learn more about registering services 
Welcome Bot Framework Emulator ->Create new bot configuration
1.Learn more 
Navigate to the Open bot ->Select the Open bot control->Open bot dialog ->Connect the bot 
1.Learn more 
Ngrok Settings
1.ngrok 2.wikipage 3.Privacy statement ..etc

**The same issue is observed, on MacOs also.**

Repro Steps.
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened 
3. Navigate to the welcome tab screen links 
4. Observe the issue 

User Impact:
Users with low vision or color-blindness will find it difficult to identify these links.

Test Environment:
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator 

OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829 (4.5.2-80829.80829))

MAS Reference Link:
[Sharepoint link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579})
](url)
![MAS1 4 1_Welcome](https://user-images.githubusercontent.com/46575396/62774420-4674eb00-bac2-11e9-80fc-1d6403f15706.png)
![Underline or chevron is not provided for links- MacOs](https://user-images.githubusercontent.com/53518994/66129136-f0797b00-e60c-11e9-82bf-faac09c85368.png)

**Comment by corinagum** [comment]
I'm not sure that this is a MAS 1.4.1 issue. In this case, color is not the sole indication of focus - the lined box is an indicator just like underlining the text would also be a way of indication.

1.4.1 doesn't seem to address using a box focus instead of underline focus. If the box is not allowed, we will need to confirm with Design that we will be changing to something else, like underline.

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue partially fixed .Hence we are reactivating this bug 
Issue still repro'ing for "1.ngrok 2.wikipage 3.Privacy statement" links and check boxes 
Please refer the attachment:1712.png
![1712](https://user-images.githubusercontent.com/53518994/65232764-546d5100-daef-11e9-8edb-780552c26e97.png)

**Comment by corinagum** [comment]
I checked this in my PR https://github.com/microsoft/BotFramework-Emulator/pull/1877, and the focus is fixed to the darker blue. I'm assuming the page got fixed sometime inbetween PR #1783 and #1877

**Comment by Kumar2608** [comment]
Issue still repro'ing for check boxes hence we are reactivating this bug.
Please refer the attachments:1712.png
![1712](https://user-images.githubusercontent.com/53518994/66129793-3420b480-e60e-11e9-897e-b9ae5ead04f6.png)

**Comment by Kumar2608** [comment]
Issue got fixed hence we are closing this bug.
